README.md

```diff
@@ -176,6 +176,13 @@ The following exceptions are thrown:
 * `RegistryAuthenticationException`: username and password are invalid
 * `RegistryAuthorizationException`: registry does not support tags using the given credentials, probably because the repository is private and the credentials provided do not have access
 
+#### digest
+````ruby
+manifest = reg.digest("namespace/repo", "2.5.6")
+````
+
+Returns the digest for the given tag of the given repository.
```

Comment:
Technically, I think it is "the digest for the manifest represented by the tag of the given repository."